src/main.js

```diff
@@ -133,6 +158,10 @@ function buildRandomPoster() {
   currentPoster = new Poster(randomImage, randomTitle, randomQuote);
 };
 
+function saveCurrentPoster(current) {
+  savedPosters.push(current);
+}
+
```

Comment:
I like this function!

Reply:
Regarding this comment -  "Also, I keep forgetting - gotta get the buildRandomPoster function to stop repeating the same selections back-to-back!" - we might be able to use a .includes to make sure the savedPosters doesn't already include a particular random object